### ISSUE TITLE
Xcode 9.3 (Swift 4.1) support

### ISSUE THREAD

**Issue by GregPerez83** [Flash3001/Xamarin.Swift#15]
Hi, the new Xcode upgraded Swift and broke compatibility with the previous swift libraries.  Would it be possible to get an update of the nuget packages?  Happy to help out in any way.

Thanks,
Greg Perez

**Comment by Flash3001** [comment]
@GregPerez83 That update is already in NuGet. The Charts library was not updated though https://github.com/Flash3001/iOSCharts.Xamarin/issues/34 (I'm working on that right now)

**Comment by GregPerez83** [comment]
Ah yes.  I didn't see a commit for it and thought it hadn't been updated.  Thanks for taking care of it!

**Comment by Flash3001** [comment]
Ops! Forgot to push. Thank you for letting me know

**Comment by MatthewGerber** [comment]
I just updated my Swift4 NuGet packages from 4.0.3 to 4.1.0. It builds fine, but the app crashes immediately upon launch both in the simulator and on a physical device. I don't get any application output, so it's hard to tell what's going wrong. Any ideas? I am attaching a screenshot of the packages I have added. One thing I noticed is that the Xamarin.Swift4 package stayed at version 4.0.0. Is this correct?

<img width="213" alt="screen shot 2018-04-16 at 10 59 48 am" src="https://user-images.githubusercontent.com/4643066/38817274-02a01340-4166-11e8-9e75-7b96ee85773f.png">
